src/frontend/actions/main-actions.ts

```diff
@@ -2,13 +2,17 @@ import {Injectable} from '@angular/core';
 import {dispatch} from '@angular-redux/store';
 import {Tab, StateTab} from '../state';
 import {IAppState} from '../store/model';
+import { Path } from '../../tree';
```

Comment:
@JinSY I don't like it either but I think the Augury project is heavily is biased towards omitting the spaces in imports (like the lines above) :). I'd like to keep it as consistent as possible.